### DISCUSSION TITLE
Invisible Modal

### DISCUSSION THREAD

**Discussion by jhoddevik** [saadeghi/daisyui#2042] [Q&A] [answered]
I'm trying to use Modal boxes in a SvelteKit project. Men when I trigger the opening with `my_modal_1.showModal()`, the modal is invisible.

I cannot click on anything else on the screen, but if I click esc it closes. So it looks like the modal is opened, it just never actually appears.

`<dialog id="my_modal_1" class="modal" open=""><form method="dialog" class="modal-box"><h3 class="font-bold text-lg">Hello!</h3> <p class="py-4">Press ESC key or click the button below to close</p> <div class="modal-action"><button class="btn">Close</button></div></form></dialog>`

**Comment by saadeghi** [comment, marked as answer]
I don't see any issues. Here's your code and it works:  
https://svelte.dev/repl/04724de054ff4543a1e6508aed8ecbbf?version=4.0.0

Are you using the latest version of daisyUI?
